utils/eclipselink.dbws.builder.test.oracle/src/it/java/dbws/testing/plsqlrecord/PLSQLTypeReturnTest.java

```diff
@@ -0,0 +1,177 @@
+package dbws.testing.plsqlrecord;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.StringReader;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+//java eXtension imports
+import javax.wsdl.WSDLException;
+
+//EclipseLink imports
+import org.eclipse.persistence.internal.xr.Invocation;
+import org.eclipse.persistence.internal.xr.Operation;
+import org.eclipse.persistence.oxm.XMLMarshaller;
+//JUnit4 imports
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+//test imports
+import dbws.testing.DBWSTestSuite;
+
+public class PLSQLTypeReturnTest extends DBWSTestSuite {
```

Comment:
Please rename this class `dbws.testing.plsqlrecord.PLSQLTypeReturnTest` should be `dbws.testing.plsqlrecord.PLSQLTypeReturnTestSuite` to match pattern `<include>**.*TestSuite*</include>` L159 in module pom.xml .
Without this, test class is not executed as a part of tests by command:
`mvn clean verify -pl :org.eclipse.persistence.dbws.builder.oracle.test -P oracle`